### PR TITLE
Fixed socket binding test case

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/SocketBindingsTestCase.java [hunk 1]
```diff
@@ -122,13 +124,13 @@ public void removeInboundSocketBinding() {
     }
 
     @Test
-    public void createOutboundLocalSocketBinding() {
+    public void createOutboundLocalSocketBinding() throws IOException, CommandFailedException {
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/SocketBindingsTestCase.java]: Why throwing those exceptions here, the code doesn't require throwing them

**Comment by honza-kasik** [comment]
Removed unnecessary exceptions, ready to be reviewed.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/SocketBindingsTestCase.java [hunk 2]
```diff
@@ -23,6 +23,9 @@
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+
+import java.io.IOException;
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/SocketBindingsTestCase.java]: You removed throwing these exceptions so you have to remove now unused imports of both these exceptions as well.

**Comment by honza-kasik** [comment]
Removed unused imports

**Comment by zebrik** [comment]
LGTM now. Thank you.

**Pull request merged**